.github/workflows/build-test.yml

```diff
@@ -50,16 +50,6 @@ jobs:
       - name: Unit Test
         run: make test
 
-      - name: Integration Test
```

Comment:
Integration test should be run as part of another job/action (otherwise it keeps failing at build time for forked projects)